flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/control/ProcessVariableSupplierPidController.java

```diff
@@ -0,0 +1,36 @@
+package com.flash3388.flashlib.robot.control;
+
+import com.flash3388.flashlib.time.Clock;
+import com.jmath.ExtendedMath;
+
+import java.util.function.DoubleSupplier;
+import java.util.function.DoubleUnaryOperator;
+
+public class ProcessVariableSupplierPidController extends PidController implements DoubleUnaryOperator {
+
+    private final DoubleSupplier mProcessVariable;
+    private final Clock mClock;
+
+    public ProcessVariableSupplierPidController(DoubleSupplier kp, DoubleSupplier ki, DoubleSupplier kd, DoubleSupplier kf, DoubleSupplier processVariable, Clock clock) {
```

Comment:
Provide overload with the robot clock: `RunningRobot.getInstance().getClock()`

Reply:
I thought about it, but it seemed not needed